.github/workflows/publish-canary.yaml

```
@@ -19,10 +21,14 @@ on:
       - 'LICENSE'
       - 'Makefile'
 
+env:
+  PUSH_TO_REGISTRIES: false
```

```suggestion
  PUSH_TO_REGISTRIES: true
```
**TODO:** This needs to be committed before merging this PR.